locales/en.default.schema.json

```diff
@@ -1983,6 +1983,182 @@
       "presets": {
         "name": "Video"
       }
+    },
+    "tabs": {
```

Comment:
@Oliviammarcello Do you mind taking a look at the content and seeing if it looks good? :) If so I believe I can already request translations

Reply:
@Oliviammarcello @sofiamatulis 
The main thing I'm not confident about is the name "Tabs" which Scott flagged (thanks!). I will get some feedback tomorrow re: some naming options (Accordion, Collapsible content etc.) and get back to you on it.